inference-engine/include/ie_transformations.hpp

```diff
@@ -52,5 +52,42 @@ namespace InferenceEngine {
  * @param network A network to apply LowLatency transformation
  * *
  */
+
+INFERENCE_ENGINE_DEPRECATED("Use InferenceEngine::LowLatency2 instead")
 INFERENCE_ENGINE_API_CPP(void) LowLatency(InferenceEngine::CNNNetwork& network);
+
+
+/**
+ * @brief The transformation finds all TensorIterator/Loop layers in the network,
+ * processes all back edges that describe a connection between Result and Parameter
+ * of the TensorIterator/Loop bodies,and inserts ReadValue and Assign layers at the
+ * input and output corresponding to this back edge.
+ * Supported platforms: CPU, GNA.
+ *
+ * The example below describes the changes made by the transformation
+ *  [] - TensorIterator body
+ *  () - new layer
+ *  BE - back-edge
+ *
+ *  before applying the transformation:
+ *  -> input1[BE_1 -> Parameter -> Layers ... -> Result  -> BE_1 ]output1->
+ *
+ *  after applying the transformation:
+ *  ->(ReadValue)-> input1[BE_1 ->Parameter->Layers ...->Result->BE_1]output1 ->(Assign)
+ *                                                                      \
+ *                                                                       ->...
+ * After applying the transformation, the resulting network can be inferred
+ * step by step, the states will store between inferences.
+ * @param network A network to apply LowLatency transformation
+ * @param use_const_initializer Changes the type of the initializing subgraph for ReadValue operations.
+          If "true", then the transformation inserts Constant before ReadValue operation.
+          If "false, then the transformation leaves existed initializing subgraph for ReadValue operation.
+ * @param sub_graph_iterations Changes a Constant (trip_count) that determines the number of iterations of
+ * Loop operation by a given number. Does not affect TensorIterators.
+ * *
+ */
+INFERENCE_ENGINE_API_CPP(void) LowLatency2(InferenceEngine::CNNNetwork& network,
```

Comment:
Maybe we can remove `2` from the function name. these two methods can be differentiated by arguments.

Reply:
We can't, because 2nd and 3rd arguments of LowLatency2 are optional